### PR TITLE
Ref: Add warning to LN

### PR DESCRIPTION
**Pull request by ncoelho**

### CODE REVIEW

**Comment by ncoelho** [comment]
Screenshot 2020-12-29 at 15 05 08

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/45m8b693enuyg9819v75zfft38

[download apk](http://filestorage.bluewallet.io:1488/1609255011.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vyzk0h0ra1q1pbnhaf5zx383f0

[download apk](http://filestorage.bluewallet.io:1488/1609261057.apk)

File: loc/en.json [hunk 1]
```diff
@@ -136,8 +136,10 @@
     "ok": "OK, I wrote this down.",
     "ok_lnd": "OK, I have saved it.",
     "text": "Please take a moment to write down this mnemonic phrase on a piece of paper. It's your backup you can use to restore the wallet on other device.",
-    "text_lnd": "Please take a moment to save this LNDHub authentication. It's your backup you can use to restore the wallet on other device.",
-    "title": "Your wallet is created."
+    "text_lnd": "Please save this wallet backup. It allows you to restore the wallet in case of loss.",
+    "text_lnd2": "Don't use it to store funds.",
```
Comment (marcosrdz) [inline loc/en.json:140]: Change Wording

**Review by marcosrdz — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/3qcadt3bbedq63db2jpbz9dm8c

[download apk](http://filestorage.bluewallet.io:1488/1609284127.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/rw35830ucdzu0d8xv16phgbvhc

[download apk](http://filestorage.bluewallet.io:1488/1609695399.apk)

**Review by marcosrdz — APPROVED** [review]

File: loc/en.json [hunk 2]
```diff
@@ -136,8 +136,10 @@
     "ok": "OK, I wrote this down.",
     "ok_lnd": "OK, I have saved it.",
     "text": "Please take a moment to write down this mnemonic phrase on a piece of paper. It's your backup you can use to restore the wallet on other device.",
-    "text_lnd": "Please take a moment to save this LNDHub authentication. It's your backup you can use to restore the wallet on other device.",
-    "title": "Your wallet is created."
+    "text_lnd": "Please save this wallet backup. It allows you to restore the wallet in case of loss.",
+    "text_lnd2": "Don't use it for large amounts.",
```
Comment (Overtorment) [inline loc/en.json:140]: i dont like this line. also, apple might object, they hate everything that hints that software is beta, for testing etc

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/q7fmaza8925ga1eqkp5cpmzxvg

[download apk](http://filestorage.bluewallet.io:1488/1609777878.apk)

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**